app/src/main/res/values-b+zh+TW/localized.xml

```diff
@@ -124,6 +124,8 @@
     <string name="p_battle_config_support_mlb">滿破禮裝</string>
     <string name="p_battle_config_support_friend_names">好友名稱</string>
     <string name="p_battle_config_support_friends_only">只選擇好友從者</string>
+    <string name="p_battle_config_support_friend_name_hint">請使用「支援圖片截取器」腳本加入好友名稱的圖片</string>
+
```

Comment:
```suggestion
    <string name="p_battle_config_support_friend_name_hint">請使用「支援圖片截取器」腳本加入好友名稱的圖片</string>
```

Hmm, the change suggestion didn't work well 😅

Reply:
Can you remove that line 128 from your side, it seems like I'm not able to suggest that from GitHub's UI.

Reply:
@MathewSachin You know, you can just click on edit in the Files changed tab and commit the blank line removal.